Src/Innovative.Blazor.Components/Components/Form/InnovativeForm.razor.css

```
@@ -87,6 +87,171 @@
     width: 100%;
 }
 
+::deep .rz-label {
+    font-weight: var(--inn-label-font-weight);
+    font-size: var(--inn-label-font-size);
+    margin-block-start: unset;
+    margin-block-end: unset;
+    margin-bottom: 0;
+}
+
+::deep .rz-colorpicker,
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)